src/pages/commits/ActiveCommits/CommitChartTooltip.tsx

```diff
@@ -10,18 +10,22 @@ import { size, zIndex } from "constants/tokens";
 
 import { TaskStatus } from "types/task";
 
+import { msToDuration } from "utils/string";
 import { ColorCount, getStatusesWithZeroCount } from "./utils";
 
 const { gray } = uiColors;
 interface Props {
   groupedTaskStats: ColorCount[];
   trigger: React.ReactElement;
+  eta?: Date;
 }
 
 export const CommitChartTooltip: React.VFC<Props> = ({
   groupedTaskStats,
   trigger,
+  eta,
 }) => {
+  const etaString = msToDuration(new Date(eta).valueOf() - Date.now());
```

Comment:
More of a general observation but it looks like `eta` is really a string instead of a Date type (since we need to create a new instance of a `Date`). I wonder if there is something that could be improved with our gqlgen/codegen config so that `Time` scalars are accurately reflected as JS Date objects.